### DISCUSSION TITLE
URL for Stellarium Constellations in comet NEOWISE sample script doesn't work anymore

### DISCUSSION THREAD

**Discussion by Michael-Lemons** [skyfielders/python-skyfield#825] [General]
"Drawing a finder chart for comet NEOWISE"
https://rhodesmill.org/skyfield/example-plots.html#drawing-a-finder-chart-for-comet-neowise

Bad URL:
url = ('https://raw.githubusercontent.com/Stellarium/stellarium/master'
       '/skycultures/western_SnT/constellationship.fab')

Replace it with this one:
https://github.com/Stellarium/stellarium/raw/master/skycultures/modern_rey/constellationship.fab

The Stellarium project on github also has constellation lines for other cultures.

**Comment by brandon-rhodes** [comment]
Thanks for pointing out that they changed paths!

891cbb96c42506c7c9d824b9736d43ca0477ebe4

The documentation will update with the next release of Skyfield.

**Comment by Michael-Lemons** [comment]
Use this comet name to plot a finder chart for the comet that has its closest approach to Earth on Feb 1, 2023.
'C/2022 E3 (ZTF)'
